Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,21 @@
 import React from 'react';
 import './Header.css';
+
+const HeaderContainer = () => {
+    return (
+      <div className = "container">
+        <div>
+          <img src="https://ibin.co/3whrpKSBbZ81.png" alt = "" className = "headerImg"/>
```

Comment:
Where this img tag is would be an appropriate place to create that ImageThumbnail component. Abstract it out into that file like you abstracted HeaderContainer and CardContainer from App.js.